presto-spi/src/main/java/com/facebook/presto/spi/StandardWarningCode.java

```diff
@@ -16,6 +16,7 @@
 public enum StandardWarningCode
         implements WarningCodeSupplier
 {
+    LEGACY_ORDER_BY(0x0000_0001)
```

Comment:
We should pick different ranges that don't conflict with error codes.

Reply:
> We should pick different ranges that don't conflict with error codes.

I thought the conclusion from our discussion a few months ago was that errors and warnings were different namespaces (there were few, if any, cases where a warning would automatically become an error. That whenever such conversion was needed, someone would have to write code for it and create an appropriate error)